packages/core/src/common/_colors.scss

```diff
@@ -7,114 +7,114 @@
 
 // Gray scale
 
-$black: #10161a !default;
-
-$dark-gray1: #182026 !default;
-$dark-gray2: #202b33 !default;
-$dark-gray3: #293742 !default;
-$dark-gray4: #30404d !default;
-$dark-gray5: #394b59 !default;
-
-$gray1: #5c7080 !default;
-$gray2: #738694 !default;
-$gray3: #8a9ba8 !default;
-$gray4: #a7b6c2 !default;
-$gray5: #bfccd6 !default;
-
-$light-gray1: #ced9e0 !default;
-$light-gray2: #d8e1e8 !default;
-$light-gray3: #e1e8ed !default;
-$light-gray4: #ebf1f5 !default;
-$light-gray5: #f5f8fa !default;
+$black: #111418 !default;
```

Comment:
these are the legacy colors, not the modern colors... you'll have to remove your changes to this file and instead update the hex values here: https://github.com/palantir/blueprint/blob/1de7170f98c7a0b17c6c6642df11a01d806f577f/packages/colors/src/_colors.scss#L10

also, our stylelint rules require you to use lowercase hex codes. if this is inconvenient when exporting colors from Figma, we can reconsider our [stylelint-config-palantir](https://github.com/palantir/stylelint-config-palantir), but that would happen as a breaking change to the config and would need to be separate from this PR. so for now you'll have to convert all these to lowercase hex codes.